scripts/limpar-marcador-nulo-nos-bens.py

```
@@ -0,0 +1,184 @@
+                nb["descricao"] = ""
+                novos.append(nb)
+                tocados += 1
+            else:
+                novos.append(b)
+        # Guarda: so a chave `descricao` pode mudar, e so onde valia o marcador.
+        antes = [{k: v for k, v in b.items() if k != "descricao"} if isinstance(b, dict) else b
+                 for b in bens]
+        depois = [{k: v for k, v in b.items() if k != "descricao"} if isinstance(b, dict) else b
+                  for b in novos]
+        if antes != depois:
+            raise SystemExit(f"ABORTA: {p['id']} mudaria campo alem de descricao")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)